examples/crowd-funding/src/lib.rs

```diff
@@ -123,7 +123,7 @@ APP_ID_0=$(linera --with-wallet 0 project publish-and-create \
            --json-argument '{ "accounts": { "User:'$OWNER_0'": "100", "User:'$OWNER_1'": "200" } }')
 
 # Wait for it to fully complete
-sleep 8
+sleep 12
```

Comment:
Any chance we can avoid this? (Perhaps boxing helped?)

Reply:
Indeed! At least it can be lower. I'll run it a few more times on my slow machine, then I'll revert or reduce it.